dmutils/email/helpers.py

```diff
@@ -2,9 +2,105 @@
+
+
+def validate_email_address(email_address):
+    """Return True if `email_address` is valid, otherwise returns False"""
+
+    # Largely a straight copy from https://github.com/alphagov/notifications-utils/blob/\
+    #   67889886ec1476136d12e7f32787a7dbd0574cc2/notifications_utils/recipients.py#L439 onwards so that we have
+    # validity-parity with Notify and minimise nasty surprises once we attempt to send an email to this address via
+    # Notify and only find out it won't be accepted once it's too late to give the user a sane validation message
+
+    # almost exactly the same as by https://github.com/wtforms/wtforms/blob/master/wtforms/validators.py,
+    # with minor tweaks for SES compatibility - to avoid complications we are a lot stricter with the local part
+    # than neccessary - we don't allow any double quotes or semicolons to prevent SES Technical Failures
+    email_address = (email_address or "").strip()
+    match = re.match(email_regex, email_address)
+
+    # not an email
+    if not match:
+        return False
+
+    hostname = match.group(1)
+    # don't allow consecutive periods in domain names
+    if '..' in hostname:
+        return False
+
+    # idna = "Internationalized domain name" - this encode/decode cycle converts unicode into its accurate ascii
+    # representation as the web uses. '例え.テスト'.encode('idna') == b'xn--r8jz45g.xn--zckzah'
+    try:
+        hostname = hostname.encode('idna').decode('ascii')
+    except UnicodeError:
+        return False
+
+    parts = hostname.split('.')
+
+    if len(hostname) > 253 or len(parts) < 2:
+        return False
+
+    for part in parts:
+        if not part or len(part) > 63 or not hostname_part.match(part):
+            return False
+
+    # if the part after the last . is not a valid TLD then bail out
+    if not tld_part.match(parts[-1]):
+        return False
+
+    return True
+
+
+def get_email_addresses(string, separators=EMAIL_ADDRESS_SEPARATORS):
+    """
+    Returns a list of email addresses from a string.
+
+    It does not validate the email addresses.
+
+    @param separators   list of characters that could be
+                        used to separate email addresses
+
+    >>> from dmutils.email.helpers import email_addresses
+    >>> email_addresses("bob@blob.com ")
+    ['bob@blob.com']
+    >>> email_addresses("bob@blob.com; bob.blob@job.com")
+    ['bob@blob.com', 'bob.blob@job.com']
+    >>> email_addresses("bob@blob.com /  bob.blob@job.com")
+    ['bob@blob.com', 'bob.blob@job.com']
+    >>> email_addresses("bob@invalid;bob.blob@job.com")
+    ['bob@invalid', 'bob.blob@job.com']
+    >>> email_addresses("bob@blob;bob.blob@job.com;bob@blob..invalid")
+    ['bob@blob', 'bob.blob@job.com', 'bob@blob..invalid']
+    """
+
+    addresses = [string]
+    for sep in separators:
+        if sep in string:
+            addresses = string.split(sep)
+            break  # earlier separators take precedence
```

Comment:
This is probably pretty unlikely, but what if the string uses different separators? `'bob@blob', 'bob.blob@job.com'; 'bob.bob@blob.com'` for example.
This is probably too much of an edge case to worry about but thought it worth thinking about.

Reply:
I thought about it but decided it was probably too much of an edge case to be worth the additional complexity.

To be fair it should be a fairly straightforward change, basically it would be

```
    for sep in separators:
        if sep in string:
            string = string.split(sep)
```

but I worried it might have weird and hard to debug side-effects.

I guess if something like this appears in the wild we can deal with it then.